apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -25,7 +25,15 @@ export function ExecutionDetailsModal({
     refetchInterval: 3000,
   });
 
-  const { jobs, _digestedNotificationId: digestedNotificationId, to: subscriberVariables } = response?.data || {};
+  const {
+    jobs,
+    _digestedNotificationId: digestedNotificationId,
+    subscriber,
+    to: subscriberVariables,
```

Comment:
I am bringing the subscriber and inside the Trigger snippet I am building it as testing it locally I never have the field `to` coming, so not sure if it is something extra I removed or we might even be better using the full information inside the field `triggers` from the `template` field.